stories/index.js

```diff
@@ -41,7 +41,67 @@ storiesOf('Picker', module)
+        emoji={text('Idle emoji', 'department_store')}
+        defaultSkin={number('Default skin tone', 1)}
+        color={color('Highlight color', '#ae65c5')}
+        showPreview={boolean('Show preview', true)}
+        showSkinTones={boolean('Show skin tones', true)}
+        custom={CUSTOM_EMOJIS}
+        notFound={{
+          notFoundImg: () => <img src='https://assets-cdn.github.com/images/icons/emoji/octocat.png?v7' />
+        }}
+      />))
+      .add('not-found-svg', () => (
+        <Picker
+          onClick={action('clicked')}
+          onSelect={action('selected')}
+          onSkinChange={action('skin changed')}
+          native={boolean('Unicode', true)}
+          set={select('Emoji pack', SETS, SETS[0])}
+          emojiSize={number('Emoji size', 24)}
+          perLine={number('Per line', 9)}
+          title={text('Idle text', 'Your Title Here')}
+          emoji={text('Idle emoji', 'department_store')}
+          defaultSkin={number('Default skin tone', 1)}
+          color={color('Highlight color', '#ae65c5')}
+          showPreview={boolean('Show preview', true)}
+          showSkinTones={boolean('Show skin tones', true)}
+          custom={CUSTOM_EMOJIS}
+          notFound={{
+            notFoundImg: () => <svg aria-labelledby="simpleicons-jira-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-jira-icon">Jira icon</title><path d="M23.323 11.33L13.001 1 12 0 4.225 7.775.67 11.33a.96.96 0 0 0 0 1.347l7.103 7.103L12 24l7.771-7.771.121-.121 3.431-3.431a.945.945 0 0 0 0-1.347zM12 15.551L8.449 12 12 8.453 15.548 12 12 15.551z"/></svg>
+          }}
+        />))
+      .add('not-found-iconfont', () => (
+        <Picker
+          onClick={action('clicked')}
+          onSelect={action('selected')}
+          onSkinChange={action('skin changed')}
+          native={boolean('Unicode', true)}
+          set={select('Emoji pack', SETS, SETS[0])}
+          emojiSize={number('Emoji size', 24)}
+          perLine={number('Per line', 9)}
+          title={text('Idle text', 'Your Title Here')}
+          emoji={text('Idle emoji', 'department_store')}
+          defaultSkin={number('Default skin tone', 1)}
+          color={color('Highlight color', '#ae65c5')}
+          showPreview={boolean('Show preview', true)}
+          showSkinTones={boolean('Show skin tones', true)}
+          custom={CUSTOM_EMOJIS}
+          notFound={{
+            notFoundImg: () => <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M4.75 9.5c0 1.795 1.455 3.25 3.25 3.25s3.25-1.455 3.25-3.25-1.455-3.25-3.25-3.25-3.25 1.455-3.25 3.25zM15 4h-3.5c-0.25-1-0.5-2-1.5-2h-4c-1 0-1.25 1-1.5 2h-3.5c-0.55 0-1 0.45-1 1v9c0 0.55 0.45 1 1 1h14c0.55 0 1-0.45 1-1v-9c0-0.55-0.45-1-1-1zM8 13.938c-2.451 0-4.438-1.987-4.438-4.438s1.987-4.438 4.438-4.438c2.451 0 4.438 1.987 4.438 4.438s-1.987 4.438-4.438 4.438zM15 7h-2v-1h2v1z"></path></svg>,
```

Comment:
Should this be a `span` with an iconfont? 
Looks liek this is pretty much the same as above.